src/main/java/org/jitsi/xmpp/mucclient/MucClientManager.java

```diff
@@ -87,29 +95,39 @@
      */
     public MucClientManager()
     {
-        this(new String[0]);
+        this(new String[0], null);
     }
 
     /**
      * Initializes a new {@link MucClientManager} instance.
      *
      * @param features the features to use for disco#info.
      */
-    public MucClientManager(String[] features)
+    public MucClientManager(String[] features, ConfigurationService cfg)
```

Comment:
why pass the whole ConfigurationService in to these ctors?  All we need in each place seems to be a string/boolean, why not just pass that in?

Reply:
I don't want to add this same logic (whether to enable smack debug) to jitsi-videobridge and later jibri/jigasi. That's how we end up not only with duplicate code, but duplicate properties

Reply:
Yes but now this class relies on ConfigurationService which is an unnecessary dependency: it only needs to know the logger class name and a boolean property.  Where those properties come from shouldn't have to rely on ConfigurationService.  Jibri, for example, doesn't use ConfigurationService but will use this class.  We don't need to pass the service in, just the resulting config itself, regardless of where it came from.  

If we had unit tests for this class you'd be feeling some of the pain of having to mock out ConfigurationService (and having to assume which property this class will check to read from its config) as opposed to just being able to pass in the desired config value.